skin.css

```diff
@@ -2,12 +2,65 @@
     background-color: #ddd;
 }
 
-button.btn {
+.btn {
+  width:170px
   border-radius: 3px;
-  border-style: solid;
-  border-width: 1px;
+  background-color:white;
   font-weight: normal;
-  line-height: 1.15;
-  text-align: center;
-  vertical-align: middle;
+  font-family:sans-serif;
+  padding: 15px;
+  border: 1px; solid lightgrey
+

```

Comment:
There three extra paragraphs are not required, can be removed for easier readability.